### PR TITLE
Mermaid Tools

### PR DESCRIPTION
**Pull request by akanex115**
# Background
https://github.com/ajktown/ConsistencyGPT/issues/29

## TODOs
- [x] Recommend: https://marketplace.visualstudio.com/items?itemName=bpruitt-goddard.mermaid-markdown-syntax-highlighting
- [x] Recommend: https://marketplace.visualstudio.com/items?itemName=bierner.markdown-mermaid

## Checklist (Developers)
- [x] `Draft` is set for this PR
- [x] `Title` is checked
- [x] `Background` is filled
- [x] `TODOs` are filled
- [x] `Assignee` is set
- [x] `Labels` are set
- [x] `development` is linked if related issue exists
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
- [x] Check if there are any other missing TODOs that are not yet listed
- [x] Review Code
- [x] Every item on the checklist has been addressed accordingly
- [x] If `development` is associated to this PR, you must check if every TODOs are handled

### CODE REVIEW

File: install/vs_code_extensions/install-ajktown-recommended-extensions.md [hunk 1]
```diff
@@ -35,4 +36,10 @@ This doc is to install the recommended extensions for AJK Town for your VS Code
 
 - VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=jebbs.plantuml
 - VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=well-ar.plantuml
-- VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=jebbs.plantuml
\ No newline at end of file
+- VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=jebbs.plantuml
```
Comment (mlajkim) [inline install/vs_code_extensions/install-ajktown-recommended-extensions.md:39]: Duplicated?
```suggestion
```

**Review by mlajkim — COMMENTED** [review]
first reivew

File: install/vs_code_extensions/install-ajktown-recommended-extensions.md (hunk 1 above)
Comment (akanex115) [reply install/vs_code_extensions/install-ajktown-recommended-extensions.md]: Sorry. Deleted it.

**Comment by mlajkim** [comment]
image

**Review by mlajkim — APPROVED** [review]
OK

**Pull request merged**